src/CVPZ.Application/Common/Behaviors/UserTracker.cs

```diff
@@ -20,7 +20,6 @@ public UserTracker(IHttpContextAccessor httpContextAccessor, IMediator mediator,
         this._mediator = mediator;
         _logger = logger;
     }
-
```

Comment:
I like empty lines in between curly braces of top level things.

Reply:
resolved :)

Reply:
resolved :)